### PR TITLE
restore: add local checksum log

### PR DESCRIPTION
**Pull request by lonng**
### What problem does this PR solve? 

Add local checksum information, which helps us to compare checksum manually when executing `ADMIN CHECKSUM TABLE table_name` timeout.

### What is changed and how it works?

Add some logs.

### Check List 

Tests 

 - Integration test

Side effects

N/A

Related changes

N/A

### CODE REVIEW

File: lightning/restore/restore.go [hunk 1]
```diff
@@ -824,12 +824,20 @@ func (t *TableRestore) postProcess(ctx context.Context, rc *RestoreController, c
 	}
 
 	// 4. do table checksum
+	var localChecksum verify.KVChecksum
+	for _, engine := range cp.Engines {
+		for _, chunk := range engine.Chunks {
+			localChecksum.Add(&chunk.Checksum)
+		}
+	}
+	common.AppLogger.Errorf("[%s] local checksum [sum:%d, kvs:%d, size:%v]",
```
Comment (kennytm) [inline lightning/restore/restore.go]:
```suggestion
	common.AppLogger.Infof("[%s] local checksum [sum:%d, kvs:%d, size:%v]",
```

Comment (kennytm) [reply lightning/restore/restore.go]: And why the size is `%v`?

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**